### PR TITLE
Add graspy

### PR DESCRIPTION
**Pull request by gmischl1**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by bdpedigo** [comment]
I am a maintainer of Graspy, ok with being listed here

**Comment by j1c** [comment]
I am also a maintainer of Graspy, and ok for being listed here

File: recipes/graspy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/neurodata/graspy/archive/v{{ version }}.tar.gz
+  sha256: 00cbd96c19113355c181bd8efa07dd487bbac1ebd9a8bf14d0da1945091a51fb
+
+build:
+  number: 0
+  skip: True  # [py<35]
```
Comment (nehaljwani) [inline recipes/graspy/meta.yaml:14]: Can this be `noarch: python` ? 

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Comment (gmischl1) [reply recipes/graspy/meta.yaml:15]: If I include` noarch: python`, the linter gives me:

"noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python"

File: recipes/graspy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/neurodata/graspy/archive/v{{ version }}.tar.gz
+  sha256: 00cbd96c19113355c181bd8efa07dd487bbac1ebd9a8bf14d0da1945091a51fb
+
+build:
+  noarch: python
+  number: 0
+  skip: True
```
Comment (nehaljwani) [inline recipes/graspy/meta.yaml:15]: Umm? That's not good :)

Comment (gmischl1) [reply recipes/graspy/meta.yaml]: oops, forgot to remove that

File: recipes/graspy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - networkx >=2.1
+    - numpy >=1.8.1
+    - scikit-learn >=0.19.1
+    - scipy >=1.1.0
+    - seaborn >=0.9.0
+    - matplotlib >=3.0.0
```
Comment (nehaljwani) [inline recipes/graspy/meta.yaml:28]: Switch to matplotlib-base here.

**Pull request merged**